### PR TITLE
tweak Run:ai entry

### PR DESCRIPTION
**Pull request by dprotaso**
Address feedback

https://github.com/knative/community/pull/1649#pullrequestreview-2504554000

### CODE REVIEW

**Review by psschwei — COMMENTED** [review]
/lgtm

File: ADOPTERS.MD [hunk 1]
```diff
@@ -31,7 +31,7 @@ If you would like to be included in this table, please open [an issue](https://g
 | ManoMano*                            |                                                                      | Development & Production | [link](https://www.manomano.es/)                                         |
 | Optum*                               |                                                                      | Production               | [link](https://optum.com/)                                               |
 | Red Hat                              | Red Hat OpenShift Serverless                                         | Generally Available      | [link](https://www.openshift.com/learn/topics/serverless)                |
-| Run:AI                               | Run:AI Platform                                                      | Production               | [link](https://www.run.ai/platform)                                      |
+| Run:ai                               | Run:AI Platform                                                      | Production               | [link](https://www.run.ai)                                               |
```
Comment (shayasoolin) [inline ADOPTERS.MD:34]: Sorry, but "Run:AI Platform" should also be changed to "Run:ai Platform". Thank you

Comment (dprotaso) [reply ADOPTERS.MD:34]: oh whoops - I didn't push my latest changes. Let me do a follow up

Comment (dprotaso) [reply ADOPTERS.MD:34]: done: https://github.com/knative/community/pull/1653

**Pull request merged**